flow/project.py

```diff
@@ -2287,21 +2397,21 @@ def _get_job_labels(self, job, ignore_errors=False):
             Dictionary with keys ``job_id``, ``labels``, and ``_labels_error``.
 
         """
-        result = {}
-        result["job_id"] = str(job)
+        result = {
+            "job_id": job.get_id(),
```

Comment:
I think we want to use `job.id` everywhere instead of `job.get_id`, as per glotzerlab/signac#382.

Reply:
I believe this also affects code in `get_aggregate_id`.

Reply:
@vyasr We currently require signac-flow compatibility back to signac 1.0. The `id` property was introduced in signac 1.3. If we could bump our signac version requirement to 1.3, that would allow us to make this change.

Reply:
I agree, don't use `str`. I would also prefer to stick to public APIs, so I think we are stuck with `get_id` until we drop version compatibility.